src/devices/sound/huc6230.cpp

```diff
@@ -0,0 +1,173 @@
+// license:BSD-3-Clause
+// copyright-holders:cam900
+/*
+    Hudson HuC6230 SoundBox
+	HuC6280 PSG with ADPCM
+
+	TODO:
+	- Volume is linear?
+	- Make it actually working
+	- Implement CDDA Volume
+*/
+
+#include "emu.h"
+#include "huc6230.h"
+
+static const int clamp(int val, int min, int max) { return std::min(max,std::max(min,val)); }
```

Comment:
This should be `constexpr`, not `const` - const-qualified return-by value has no effect.